app/pages/project/instances/instance/tabs/StorageTab.tsx

```diff
@@ -112,7 +120,7 @@ export function StorageTab() {
   const { data: instance } = usePrefetchedApiQuery('instanceView', instancePathQuery)
 
   const makeActions = useCallback(
-    (disk: Disk): MenuAction[] => [
+    ({ disk, instance }: InstanceDisk): MenuAction[] => [
```

Comment:
By making the instance part of the row data (same instance for all rows) we can avoid having to remake this callback whenever the instance state changes. This works, but I can't help feeling there has to be a better way.